walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/resume.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>The One with A Resume</title>
+
+  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+  <h1> This will eventually have content, but for now this is my resume page!</h1>
+
+  <ul>
+        <li><a class="resume-snippet" id="tech" href="#"></a></li>
+        <li><a class="resume-snippet" id="exp" href="#"></a></li>
+        <li><a class="resume-snippet" id="acad" href="#"></a></li>
+        <li><a class="resume-snippet" id= "lead" href="#"></a></li>
+	</ul>
+</body>
+</html>
```

Comment:
nit: add a newline at end of code files.

Reply:
Resolved!